src/job/jobHandler.ts

```diff
@@ -401,6 +401,12 @@ export abstract class JobHandler {
       envVars += `PATH_PREFIX=${pathPrefix}\n`;
     }
     const snootyFrontEndVars = {
+      // Setting this to always be true. It's one less SSM parameter to manage,
+      // and it doesn't hurt to always do the filtering. The only reason
+      // it exists here is so that we don't always filter when testing locally.
+      // This is because the filter functionality updates the ComponentFactory.js file
+      // which could be accidentally committed if run locally.
+      USE_FILTER_BRANCH: true,
```

Comment:
Will there be an easy way to flip this variable if something goes wrong in prod though? Just wondering on a quick roll-back plan.